pkg/filesystem/file.go

```diff
@@ -14,22 +14,21 @@
 package filesystem
 
 import (
-	"io"
-	"os"
-	"path/filepath"
-
 	oktetoLog "github.com/okteto/okteto/pkg/log"
 	"github.com/spf13/afero"
+	"io"
+	"os"
 )
 
-func FileExistsWithFilesystem(name string, fs afero.Fs) bool {
-	_, err := fs.Stat(name)
+// FileExistsWithFilesystem return true if the file exists
+func FileExistsWithFilesystem(path string, fs afero.Fs) bool {
```

Comment:
Nit: Not blocker at all.

Would it make sense to create an struct `FilsystemChecker` or similar with the fs as dependency? It seems odd (as a client) to pass the filesystem to the function

Reply:
Agree, this way we could instantiate it at the beginning of the main file and pass it only when it's needed without the need of initialising the `NewOsFs` all along the code

Reply:
sounds good